docs/source/notes/cuda.rst

```diff
@@ -201,20 +201,42 @@ ensure proper synchronization.
 Stream semantics of backward passes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Internally, each backward CUDA op runs on the same stream that was used for its corresponding forward op.
-
-When manually supplying CUDA tensor(s) as a backward pass's initial gradient(s) (e.g.,
+A. Each backward CUDA op runs on the same stream that was used for its corresponding forward op.
```

Comment:
Should we talk about device here?
The current device will be the device of the output of the op. If the first output of the op is not a cuda Tensor, the current device will be the same as the caller's one.